.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: ruby
+sudo: false
```

Comment:
This is wrong. `sudo` is used because `rabbitmqctl` is used by `bin/ci/before_script`.